src/modules/settingsPage.js

```diff
@@ -298,25 +298,20 @@ exportModule({
 		}
 		let blockSettings = create("div");
 		let blockInstructions = create("p",false,false,blockSettings);
-		blockInstructions.innerText = `
-Block stuff in the home feed.
-
-Example1: To block "planning" activities by a specific user, fill out those two fields and leave the media field blank.
-Example2: To block a specific piece of media, fill out that field and leave the other two blank.
-`;
+		blockInstructions.innerText = translate("$settings_blockInstructions");
 		let blockInput = create("div","#blockInput",false,blockSettings);
-		create("span",false,"User: ",blockInput);
+		create("span",false,translate("$settings_blockUser") + " ",blockInput);
 		let blockUserInput = create("input",false,false,blockInput,"width:100px;margin-right:10px;");
 		blockUserInput.value = "";
-		create("span",false," Status: ",blockInput);
+		create("span",false," " + translate("$settings_blockStatus") + " ",blockInput);
 		let blockStatusInput = create("select",false,false,blockInput,"margin-right:10px;");
-		const blockStatuses = ["","all","status","progress","anime","manga","planning","watching","reading","pausing","dropping","rewatching","rereading","rewatched","reread"];
+		const blockStatuses = ["","all","status","progress","anime","manga","planning","watching","reading","pausing","dropping","rewatching","rereading","rewatched","reread"]; //TODO: We should also find a way to localize these
```

Comment:
Probably something like making a bunch of "$blockStatus_*" keys

Reply:
Should we also add a $blockStatus_ key? Otherwise 
![this](https://user-images.githubusercontent.com/35269770/214146875-72d8dd8e-fb56-415a-ba76-b950118b1d87.png) will show up

Reply:
Nah, ``(status ? capitalize(translate("$blockStatus_" + status)) : "")``